gyrinx/core/static/core/scss/styles.scss

```
@@ -35,6 +35,22 @@ $cyan: #10bdd3;
 // Card header background — match section header bars (bg-body-tertiary)
 $card-cap-bg: var(--bs-tertiary-bg);
 
+// The section-header / card-header surface. Bootstrap's light-mode default
+// ($gray-100, #f8f9fa) is 3% off white, so those bars read as barely-there
+// against the page; one step darker makes a section header actually separate its
+// content from what precedes it. Dark mode is a different variable
+// ($body-tertiary-bg-dark) and keeps Bootstrap's value, where contrast was
+// already right.
+//
```

Stylelint (17.14.0)

[error] Unexpected empty comment (scss/comment-no-empty)

(scss/comment-no-empty)